standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java

```diff
@@ -4154,18 +4209,14 @@ private String getDefaultPartitionName(String inputDefaultPartName) {
    * @param result The resulting names.
    * @return Whether the result contains any unknown partitions.
    */
-  private boolean getPartitionNamesPrunedByExprNoTxn(String catName, String dbName, String tblName, List<FieldSchema> partColumns, byte[] expr,
```

Comment:
nit: though it looks a bit un-formatted, I would prefer to keep it, this cloud help code review and easy to track via git history.